specificatie/openapi.yaml

```diff
@@ -204,362 +203,160 @@ paths:
-        '406':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/406"
-        '500':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/500"
-        '501':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/501"
-        '503':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/503"
-        'default':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/default"
-      tags:
-        - Ingeschreven Personen
-  /ingeschrevenpersonen/{burgerservicenummer}/partners/{id}:
-    get:
-      deprecated: true
-      operationId: GetPartner
-      summary: "Raadpleeg de partner van een persoon"
-      description: |
-                    Raadpleeg de partner van een persoon
-      parameters:
-        - $ref: '#/components/parameters/burgerservicenummer'
-        - in: path
-          name: id
-          description: |
-                        De identificatie van de partner.
-          required: true
-          schema:
-            type: string
+        Default krijg je personen terug die nog in leven zijn, tenzij je de inclusiefoverledenpersonen=true opgeeft.
+
+
+        Het is verplicht om met de fields parameter de gewenste velden in het antwoord te definieren, [zie functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.1.0/features/fields_extensie.feature)
+
+
+        1.  Persoon
+            -  geboorte__datum
+            -  naam__geslachtsnaam (minimaal 2 karakters, [wildcard toegestaan](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature)
+
+
+        2.  Persoon
+            -  verblijfplaats__gemeenteVanInschrijving
+            -  naam__geslachtsnaam (minimaal 2 karakters, [wildcard toegestaan](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature)
+
+
+        3.  Persoon
+            -  burgerservicenummer
+
+
+        4.  Postcode
+            -  verblijfplaats__postcode
+            -  verblijfplaats__huisnummer
+
+
+        5.  Straat
+            -  verblijfplaats__straat (minimaal 2 karakters, [wildcard toegestaan](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature) )
+            -  verblijfplaats__gemeenteVanInschrijving
+            -  verblijfplaats__huisnummer
+
+
+        6.  Adres
+            -  verblijfplaats__nummeraanduidingIdentificatie
+      requestBody:
+        content:
+          application/hal+json:
```

Comment:
Dit moet application/json zijn. Er zitten geen HAL elementen in de request body

Reply:
Aangepast